dashboard/test/lib/user_multi_auth_helper_test.rb

```diff
@@ -20,13 +20,17 @@ class UserMultiAuthHelperTest < ActiveSupport::TestCase
   test 'oauth_tokens_for_provider returns most recently updated tokens for migrated teacher' do
```

Comment:
this class does not appear to be using `use_transactional_test_case`. any reason why not?

Reply:
I'm not particularly familiar with this test. Why do you think `use_transactional_test_case` would be relevant?

Reply:
Just because we are running into problems with duplicate emails, and wrapping the test in a transaction would cause it not to persist and changes to the DB, which could decrease the changes of conflicting emails. It sounds from your earlier comment that this test isn't failing and isn't even using the same DB as the test that is failing though, because this is a unit test and it is a UI test that is failing. so with that information I don't think this could be causing a UI test to fail.

Reply:
Ah, that makes sense. Yeah, this test was touched because of the original PR's changes to the way we handle duplicate authentication option detection, not because of the email issue